app/Http/Controllers/ResultController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Result;
+use Illuminate\Http\Request;
+use App\Http\Resources\Answer;
+
+class ResultController extends Controller
```

Comment:
I think this should be AnswerController as the only method in it returning the answers for the given result.